MonoGame.Framework/Audio/Xact/Cue.cs

```diff
@@ -81,6 +81,11 @@ public bool IsPreparing
         }
 
         /// <summary>Returns whether the cue is prepared to play.</summary>
+        /// <remarks>
+        /// This property returns <see langword="true"/> only if the cue is prepared but has not yet been played.
+        /// Cues that are played return <see langword="true"/> for <see cref="IsPlaying"/> if they are currently playing
+        /// or <see langword="true"/> for <see cref="IsStopped"/> if they are stopped.
+        /// </remarks>
```

Comment:
Thanks for the extra effort here @Maniekko .

First i would avoid writing docs about other members in this one... like we don't need to document what IsPlaying and IsStopped does unless it directly relates to the behavior of this one.

I would instead say something like:

```
This returns <see langword="true"/> after the cue has been created, has finished preparing, and is ready to be played for the first time.
```